jr-objects/src/main/java/com/fasterxml/jackson/jr/ob/impl/BeanPropertyIntrospector.java

```diff
@@ -68,6 +68,8 @@ private POJODefinition _introspectDefinition(Class<?> beanType,
                     } else if (argType == Long.class || argType == Long.TYPE) {
                         constructors.addLongConstructor(ctor);
                     }
+                } else if (RecordsHelpers.isRecordConstructor(beanType, ctor, propsByName)) {
```

Comment:
I think this needs to come before other checks, so as not to miss single-Long Record constructor?

Reply:
Done.